data/campaigns/Northern_Rebirth/scenarios/08a_Elvish_Princess.cfg

```diff
@@ -30,7 +30,7 @@
             story= _ "One day a gryphon came screeching into the caves with the news that a large elvish force approached from the east. Messages were quickly dispatched to the elves asking their purpose. The haughty reply was that their purpose was none of any dwarf’s or human’s business, and that all would be well-advised to stay in their holes when the elves passed near."
         [/part]
         [part]
-            story= _ "Before the humans and dwarves could respond to this message, another gryphon arrived with the news that the orcs were holding an elvish sorceress prisoner in the castle of Bitterhold, less than a day’s march distant to the northwest. The castle had attracted notice because it had been recently rebuilt and manned by a unit known to include some of the hardiest and toughest of veteran orcs."
+            story= _ "Before the humans and dwarves could respond to this message, another gryphon arrived with the news that the orcs were holding an elvish sorceress prisoner in the castle of Bitterhold, less than a day’s march to the northwest. The castle had attracted notice because it had been recently rebuilt and manned by a force known to include some of the hardiest and toughest of veteran orcs."
```

Comment:
I take your point about unit having a different meaning in terms of game-play, but I prefer the (presumably) original meaning of a military unit/squad.

Reply:
Anyone else's thoughts on this? I like the new one. OR we could change the word if anyone has a better one.

Reply:
I usually don't think of a horde as a unit, just a large unorganized rabble...